salt/flink/templates/flink.conf.tpl

```diff
@@ -229,10 +229,14 @@ historyserver.archive.fs.refresh-interval: 10000
 # Configure log directory path for PNDA flink
 env.log.dir: /var/log/pnda/flink
 python.dc.tmp.dir: {{ namenode }}/{{ python_tmp_dir }}/dc
+
 # Expose Metric to an external system (Graphite) by configuring the Reporter
 metrics.reporters: grph
 metrics.reporter.grph.class: org.apache.flink.metrics.graphite.GraphiteReporter
 metrics.reporter.grph.host: {{ graphite_node }}
 metrics.reporter.grph.port: {{ graphite_port }}
 metrics.reporter.grph.protocol: TCP
 metrics.reporter.grph.prefix: flink
+
+# Configure Metric reporting interval
+metrics.reporter.grph.interval: 1 SECONDS
```

Comment:
Why is this reporting every second if the retention frequency is 60 seconds?

Reply:
This was done to see more interactive charts in grafana - due to more data points.
Have updated it to the 1 minute now.